.github/ISSUE_TEMPLATE.md

```diff
@@ -1,24 +1,28 @@
-Thanks for submitting an issue! 
+Thanks for submitting an issue! Please fill in as much of the template below as
+you can.
 
--------------BUG REPORT --------------------
-1. Fill in as much of the template below as you can. 
+------------- BUG REPORT TEMPLATE --------------------
 
-2. What kops version are you running? use `kops version`
+1. What kops version are you running? Use `kops version`?
```

Comment:
I would change the "Use `kops version`?" to something like: 

(run `kops version`)

Reply:
Concur.  This isn't a question.